uktrade/data-hub-frontend
# Add call to action to the remaining filters chips

## Description of change
As we now have more filters we now have more filter chips appearing in the collection list header. I have now added call to actions on all the filter chips so you can now remove these filters in the same way as you could with the adviser filter chips.

Note: There are clear commonalities all the filter chips but these will be addressed in the next pull request.

## Test instructions
Go to the investments projects collection page and apply new filters, once the filters are applied you should be able to clear them by clicking on the chips.

## Screenshots
### Before
![Screenshot 2020-11-25 at 13 19 59](https://user-images.githubusercontent.com/10154302/100232933-f0c78a80-2f20-11eb-8453-ec9d5069d7b8.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
